tests/akash/matrix.futil

```diff
@@ -0,0 +1,160 @@
+import "primitives/core.futil";
+import "primitives/binary_operators.futil";
+import "primitives/memories/comb.futil";
+
+// 4x4 matrix multiply
+component main(@go go: 1) -> (@done done: 1) {
+  cells {
+    @external A = comb_mem_d2(32, 4, 4, 3, 3);
+    @external B = comb_mem_d2(32, 4, 4, 3, 3);
+    @external C = comb_mem_d2(32, 4, 4, 3, 3);
```

Comment:
Sorry, I totally failed to tell you this, but we're encouraging people to use `seq_mem_d*` instead of `comb`. See more [here](https://docs.calyxir.org/libraries/core.html#memories)!